### ISSUE TITLE
Cant access demo

### ISSUE THREAD

**Issue by tebbbb** [notclickable-jordan/starbase-80#34]
I am living in Switzerland and i get

Sorry, you have been blocked
You are unable to access starbase80.dev

I just heard about your project for the first time 10min ago and wanted to check out the demo.

**Comment by JanCBredow** [comment]
same here

**Comment by NeoMod** [comment]
same.

**Comment by notclickable-jordan** [comment]
Yeah, sorry about that. I'm running my personal site from a tiny homelab and can't handle all the international traffic. So I made a copy of it on GitHub pages so you can see what the code looks like in your browser.
